.github/workflows/build-wheels-aarch64-rknn.yaml

```
@@ -42,14 +42,12 @@ jobs:
 
       - name: Build sherpa-onnx
-        uses: addnab/docker-run-action@v3
-        with:
-            image: quay.io/pypa/manylinux_2_28_aarch64
-            # image: quay.io/pypa/manylinux2014_aarch64 # it does not provide GLIBCXX 3.4.21+
-            options: |
-              --volume ${{ github.workspace }}/:/k2-fsa/sherpa-onnx
-            shell: bash
-            run: |
+        shell: bash
+        run: |
```

actionlint (1.7.10)

[error] property "config" is not defined in object type {os: string; python-version: number}

(expression)